### ISSUE TITLE
meteor build doesn't support node: protocol imports

### ISSUE THREAD

**Issue by markdowney** [meteor/meteor#11835]
Seems like [`node: protocol`](https://nodejs.org/api/esm.html#esm_node_imports) imports are not supported.

Steps to reproduce:
- create new app
- add `node-fetch` npm module and import in code `import fetch from "node-fetch";`
- run app

Result:
```
Unable to resolve some modules:

  "node:http" in /app/node_modules/node-fetch/src/index.js (os.osx.x86_64)
  "node:https" in /app/node_modules/node-fetch/src/index.js (os.osx.x86_64)
  "node:zlib" in /app/node_modules/node-fetch/src/index.js (os.osx.x86_64)
  "node:stream" in /app/node_modules/node-fetch/src/index.js (os.osx.x86_64)
  "node:util" in /app/node_modules/node-fetch/src/body.js (os.osx.x86_64)
  "node:fs" in /app/node_modules/fetch-blob/from.js (os.osx.x86_64)
  "node:path" in /app/node_modules/fetch-blob/from.js (os.osx.x86_64)
  "node:worker_threads" in /app/node_modules/fetch-blob/from.js (os.osx.x86_64)
  "node:url" in /app/node_modules/node-fetch/src/request.js (os.osx.x86_64)

If you notice problems related to these missing modules, consider running:

  meteor npm install --save node:http node:https node:zlib node:stream node:util node:fs node:path node:worker_threads node:url
```

**Comment by maxmatthews** [comment]
I'm having the same issue with the [tempy](https://www.npmjs.com/package/tempy) package. It works fine, but every time I start meteor I get:
```
Unable to resolve some modules:

  "node:fs" in /Users/maxmatthews/Desktop/projects/node_modules/tempy/index.js (os.osx.x86_64)
  "node:path" in /Users/maxmatthews/Desktop/projects/node_modules/tempy/index.js (os.osx.x86_64)
  "node:stream" in /Users/maxmatthews/Desktop/projects/node_modules/tempy/index.js (os.osx.x86_64)
  "node:util" in /Users/maxmatthews/Desktop/projects/node_modules/tempy/index.js (os.osx.x86_64)

If you notice problems related to these missing modules, consider running:

  meteor npm install --save node:fs node:path node:stream node:util
```
even though the package works fine. All because the package imports things using the `node:` protocol 
```import fs, {promises as fsPromises} from 'node:fs';
import path from 'node:path';
import stream from 'node:stream';
import {promisify} from 'node:util';
```

**Comment by Grubba27** [comment]
Hello there, hope you all are doing well :)
this issue was solved in [2.7](https://docs.meteor.com/changelog.html#v2720220324:~:text=Adds%20support%20for%20node%3A%20imports.)
I'm closing this issue
